packages/block-editor/src/components/inspector-controls/index.js

```diff
@@ -9,10 +9,19 @@ import { createSlotFill } from '@wordpress/components';
 import { ifBlockEditSelected } from '../block-edit/context';
 
 const { Fill, Slot } = createSlotFill( 'InspectorControls' );
+const { Fill: FillButtons, Slot: SlotButtons } = createSlotFill( 'InspectorControlsButtons' );
 
-const InspectorControls = ifBlockEditSelected( Fill );
+const InspectorControlsFill = ( { buttons, ...rest } ) => {
+	if ( buttons ) {
+		return <FillButtons { ...rest } />;
+	}
+	return <Fill { ...rest } />;
+};
+
+const InspectorControls = ifBlockEditSelected( InspectorControlsFill );
 
 InspectorControls.Slot = Slot;
+InspectorControls.SlotButtons = SlotButtons;
```

Comment:
Define separate slots for `Buttons` allows to render button to manage the open/close state of the `BottomSheet` and also render the whole content (`Fill` component) of this panel using the same markup: 

`<InspectorControls>
    {content to render here}
</InspectorControls>
`